starboard/common/test_main.cc

```diff
@@ -27,12 +27,19 @@ int InitAndRunAllTests(int argc, char** argv) {
 }
 }  // namespace
 
-// When we are building Evergreen we need to export SbEventHandle so that the
-// ELF loader can find and invoke it.
+#if BUILDFLAG(IS_STARBOARD)
+// For the Starboard OS define SbEventHandle as the entry point
 SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(InitAndRunAllTests);
+
+#if !SB_IS(EVERGREEN)
+// Define main for non-Evergreen Starboard OS.
+int main(int argc, char** argv) {
+  SbRunStarboardMain(argc, argv, SbEventHandle);
+}
+#endif  // !SB_IS(EVERGREEN)
 #else
+// If the OS is not Starbaord use the regular main e.g. ATV
 int main(int argc, char** argv) {
-  ::testing::InitGoogleTest(&argc, argv);
-  return RUN_ALL_TESTS();
+  InitAndRunAllTests(argc, argv);
```

Comment:
nit: `return InitAndRunAllTests(...);` (missing `return`). Same in l.37.

Reply:
Done